src/main/java/teammates/ui/controller/InstructorFeedbackRemindParticularStudentsPageData.java

```diff
@@ -1,14 +1,43 @@
 package teammates.ui.controller;
 
+import java.util.List;
+import java.util.ArrayList;
 import teammates.common.datatransfer.AccountAttributes;
 import teammates.common.datatransfer.FeedbackSessionResponseStatus;
+import teammates.ui.template.RemindParticularStudentsCheckboxEmailNamePair;
 
 public class InstructorFeedbackRemindParticularStudentsPageData extends PageData {
     public FeedbackSessionResponseStatus responseStatus;
     public String courseId;
     public String fsName;
+    private List<RemindParticularStudentsCheckboxEmailNamePair> emailNamePairs;
     
     public InstructorFeedbackRemindParticularStudentsPageData(AccountAttributes account) {
         super(account);
     }
+    
+    public void init() {
+        emailNamePairs = new ArrayList<RemindParticularStudentsCheckboxEmailNamePair>();
+
+        for (String studentEmail : responseStatus.noResponse) {
+            String studentName = responseStatus.emailNameTable.get(studentEmail);
+            
+            RemindParticularStudentsCheckboxEmailNamePair emailNamePair = 
+                new RemindParticularStudentsCheckboxEmailNamePair(studentEmail, studentName);
+            
+            emailNamePairs.add(emailNamePair);
+        }
```

Comment:
This can be replaced by a `Map`.
Just use `responseStatus.noResponse` with `responseStatus.emailNameTable`?
On the JSTL side, you should be able to use it like this (assuming you have getters for them):
`emailNameTable[studentEmail]` => `studentName`